regression/goto-cc-goto-analyzer/reachable-functions-export-file-local-symbols/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+test.c
+--reachable-functions
+^.* foo 1 4$
+^EXIT=0$
+^SIGNAL=0$
+--
+--
```

Comment:
Could we have a regex for the mangled name that would result if `base_name` was printed?

Reply:
Done!

Reply:
Thanks!